src/einsteinpy/symbolic/ricci.py

```diff
@@ -85,6 +87,60 @@ def from_metric(cls, metric):
         rc = ChristoffelSymbols.from_metric(metric)
         return cls.from_christoffels(rc)
 
+    def change_config(self, newconfig="uu"):
```

Comment:
I would say you have developed `change_config` on the lines of `MetricTensor`, however you should have done them similar to `RiemannCurvatureTensor`.

In fact, change `from_riemann`, `from_metric`, `from_christoffels` according to `RiemannCurvatureTensor`